modules/boards/app/views/boards/boards/overview.html.erb

```diff
@@ -0,0 +1,51 @@
+<%#-- copyright
+OpenProject is an open source project management software.
+Copyright (C) 2012-2023 the OpenProject GmbH
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License version 3.
+
+OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+Copyright (C) 2006-2013 Jean-Philippe Lang
+Copyright (C) 2010-2013 the ChiliProject Team
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+See COPYRIGHT and LICENSE files for more details.
+
+++#%>
+
+<% html_title(t('boards.label_boards')) -%>
+
+<%= toolbar title: t('boards.label_boards') do %>
+  <% if authorize_for(:boards, :new) %>
+    <li class="toolbar-item">
+      <a href="<%= "foo" %>"
+         id="add-board-button"
+         title="<%= I18n.t(:label_meeting_new) %>"
+         arial-label="<%= I18n.t(:label_meeting_new) %>"
+         class="button -alt-highlight">
+        <%= op_icon('button--icon icon-add') %>
+        <span class="button--text"><%= t(:label_meeting) %></span>
+      </a>
+    </li>
+  <% end %>
+<% end %>
```

Comment:
The modal for creating a new board currently does not include any selection of the project. Should we add a WP (like we have for global meetings create) to implement a new modal view that will include a product selection?

//cc @oliverguenther

Reply:
Never mind, I saw that https://community.openproject.org/projects/openproject/work_packages/47989/activity exists